app/controllers/patients/edit_controller.rb

```diff
@@ -34,6 +37,39 @@ def update_nhs_number_merge
     end
   end
 
+  def edit_school
+    render :school
+  end
+
+  def update_school
+    if school_id == "home_schooled"
+      @patient.home_educated = true
+      @patient.school_id = nil
+    elsif school_id == "unknown"
+      @patient.home_educated = false
+      @patient.school_id = nil
+    else
+      @patient.home_educated = nil
+      @patient.school_id = school_id.presence
+    end
+
+    redirect_to edit_patient_path(@patient) and return unless @patient.changed?
+
+    if @patient.save
```

Comment:
I'm not sure we should be saving the patient here, I think we should leave changing the values of `home_educated` and `school_id` to the sole responsibility of `SchoolMove`.


```suggestion
    if @patient.valid?
```